tests/bot-entrypoint.py

```diff
@@ -28,10 +28,10 @@ def load_env():
         "echo yes | bin/keybaseca backup > /shared/cakey.backup\n"
         # The output from this sign operation is tested in test_env_1.py
         "ssh-keygen -t ed25519 -f /shared/userkey -N '' && bin/keybaseca sign --public-key /shared/userkey.pub > /shared/keybaseca-sign.out\n"
-        "bin/keybaseca service &"
+        "bin/keybaseca --debug service > /tmp/debug.out &"
     ) % (shlex.quote(path)))
     # Sleep so keybaseca has time to start
-    time.sleep(5)
+    time.sleep(10)
```

Comment:
Running the tests on my (slow) laptop had some intermittent failures with this, so just bumped it. It makes the tests a little slower but they're already slow enough so /shrug